source/mainnet/net/guides/run-local-chain.rst

```diff
@@ -0,0 +1,185 @@
+
+.. code-block:: console
+
+    $ protoc --version
+    libprotoc 3.21.12
+    $ rustc --version
+    rustc 1.68.2 (9eb3afe9e 2023-03-27)
+
+.. Note::
+
+    The recommended method to install Rust is using ``rustup``. After installing ``rustup``, the Rust toolchain is then easily installed by issuing ``rustup toolchain install 1.68``.
+
+To build the ``genesis-creator`` tool, first clone the ``concordium-misc-tools`` repository which contains the source and check out the ``git`` sub-modules in the ``genesis-creator`` directory:
+
+.. code-block:: console
+
+    git clone git@github.com:Concordium/concordium-misc-tools.git
+    cd genesis-creator
+    git submodule update --init --recursive
+
+The project is then built using ``cargo``:
+
+.. code-block:: console
+
+    cargo build --release
+
+This produces the binary ``./target/release/genesis-creator`` which you run to generate the genesis data.
+
+Running the tool
+----------------
+
+The ``genesis-creator`` tool uses a TOML configuration file format for specifying parameters from which the genesis data is generated. The TOML file specifies
+
+* the initial protocol version
+* cryptographic parameters
+* anonymity revokers
+* identity providers
+* foundation accounts
+* keys for updating the chain
+* various parameters for the genesis
+
+Furthermore, it specifies where to save the output that is used to invoke the node binary. Many of these options are not relevant when testing smart contracts and the easiest way to get started is to piggyback on one of the examples in the ``./examples`` folder. In the following, you will use the file ``./examples/genesis5.toml`` and modify it slightly. Inspecting the file reveals that it specifies an initial protocol version of 5, to output credentials for 5 baker accounts, 2 foundation accounts, and the genesis time set to the system time at generation. It also specifies 5 seconds as the average time per block. Further inspection of the table at the ``accounts`` key of the file reveals that the bakers each have an initial balance of 10^15 microCCD and a stake of 5 * 10^14 microCCD. Change this section such that only one baker credential is produced by setting the value of the ``repeat`` key to 1 as follows:
+
+.. code-block:: toml
+
+    [[accounts]]
+    kind = "fresh"
+    balance = "1000000000000000"
+    stake = "500000000000000"
+    template = "baker"
+    identityProvider = 0
+    numKeys = 1
+    threshold = 1
+    repeat = 1 # Changed from 5
+
+.. Note::
+
+    Note that the staked amount needed to participate in the finalization committee is some fraction of the total amount of existing CCD defined by the value of the ``capitalBound`` key in the configurations file. The total amount is the sum of the balances of all the baker and foundation accounts specified in the genesis configuration file. In this particular example, the stake is sufficient for baking.
```

Comment:
I'm not sure it's important to mention the more fine grained chain parameters. See my comment above.

Reply:
Yea, let's figure out what to do in the above, and then change this accordingly.